.github/workflows/osgeo4w.yml

```diff
@@ -127,15 +127,49 @@ jobs:
         run: |
           python -m pip install ^
             pytest ^
+            pytest-randomly ^
             pytest-timeout
         shell: cmd /D /E:ON /V:OFF /S /C "CALL C:/OSGeo4W/OSGeo4W.bat "{0}""
-      - name: Run pytest with a single worker
+      - name: Run pytest with a single worker I
+        continue-on-error: true
         run: |
           call %OSGEO4W_ROOT%\opt\grass\etc\env.bat
           set PYTHONPATH=%GISBASE%\etc\python;%PYTHONPATH%
           path %GISBASE%\lib;%GISBASE%\bin;%PATH%
           pytest ^
             @.github/workflows/pytest_args_ci.txt ^
+            --maxfail=10 ^
+            -k "not testsuite"
+        shell: cmd /D /E:ON /V:OFF /S /C "CALL C:/OSGeo4W/OSGeo4W.bat "{0}""
+      - name: Run pytest with a single worker II
+        continue-on-error: true
+        run: |
+          call %OSGEO4W_ROOT%\opt\grass\etc\env.bat
+          set PYTHONPATH=%GISBASE%\etc\python;%PYTHONPATH%
+          path %GISBASE%\lib;%GISBASE%\bin;%PATH%
+          pytest ^
+            @.github/workflows/pytest_args_ci.txt ^
+            --maxfail=10 ^
+            -k "not testsuite"
+      - name: Run pytest with a single worker III
+        continue-on-error: true
+        run: |
+          call %OSGEO4W_ROOT%\opt\grass\etc\env.bat
+          set PYTHONPATH=%GISBASE%\etc\python;%PYTHONPATH%
+          path %GISBASE%\lib;%GISBASE%\bin;%PATH%
+          pytest ^
+            @.github/workflows/pytest_args_ci.txt ^
+            --maxfail=10 ^
+            -k "not testsuite"
```

Comment:
These were missing the shell: instruction, so didn't run.

Maybe since I didn't see anything to clear the GISRC or the hypothesis on what was the one-time change that made the rest work, only the first iteration we can conclude.

Reply:
The first two actually had shell. Now all of them do.

I actually wanted to follow up on the GISRC. What do you mean by cleaning it? Nothing should remain between the steps.